modules/ossm-cert-manager-install-istio-resource.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-istio-resource_{context}"]
+= Install your Istio resource
+
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+
+After you have installed `istio-csr` by following the procedure for either `InPlace` or `RevisionBased`, you can install the `Istio` resource.
+
+This disables Istio's built in CA server and points istiod to the `istio-csr` CA server. The `istio-csr` CA server issues certificates for both istiod and user workloads. Additionally, the istiod TLS cert is mounted into the pod at a known location where it will be read. is mounted in a known location where it will be read by istiod. Mounting the certificates to a known location is only necessary for {SMProduct}.
+
+.Procedure
+
+. Create the `Istio` object as in the following example:
++
+.Example `Istio` object
+[source, yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.23.0
+  namespace: istio-system
+  values:
+    global:
+      caAddress: cert-manager-istio-csr.cert-manager.svc:443
+    pilot:
+      env:
+        ENABLE_CA_SERVER: "false"
+      volumeMounts:
+        - mountPath: /tmp/var/run/secrets/istiod/tls
+          name: istio-csr-dns-cert
+          readOnly: true
+----
++
+[NOTE]
+====
+If you followed the instructions for "istio-csr installation: RevisionBased update strategy", then you need to add the following to your `Istio` object yaml:
+
+[source, yaml]
+----
+kind: Istio
+metadata:
+  name: default
+spec:
+  updateStrategy: RevisionBased
+----
+====
+
+. Create the `Istio` resource using the following command:
++
+[source, terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+. Wait for `Istio` to become ready:
++
+[source, terminal]
+----
+$ oc wait --for=condition=Ready istios/default -n istio-system
+----
+
+.Verification
+
+Use the sample `httpbin` service and `sleep` app to check traffic between the workloads is possible and check the workload certificate of the proxy to verify that the cert-manager tool is installed correctly.
+
+. Create the `sample` namespace:
++
+[source, terminal]
+----
+$ oc new-project sample
```

Comment:
Maybe the example that creates `istio-system` should use `new-project` instead? Either way both should probably be consistent.